src/components/PublicHeader.tsx

```diff
@@ -0,0 +1,47 @@
+interface PublicHeaderProps {
+    userData: {
```

Comment:
In the `OrgHeader` component, the equivalent property was called `org`, but here it's `userData`. Why the `Data` suffix?

I think either is fine, and would just prefer if we use the same naming pattern concistently.

Reply:
I agree! Will change this.